crm_location/views/crm_lead_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record id="view_crm_case_form_view_leads" model="ir.ui.view">
+            <field name="name">crm.lead.form.city</field>
+            <field name="model">crm.lead</field>
+            <field name="inherit_id" ref="crm.crm_case_form_view_leads" />
+            <field name="arch" type="xml">
+                <field name="street2" position="after">
```

Comment:
I would put before the city field instead of after the street2, because there is for example another module that adds street3 to the model, and can provoke an ugly view